extension/source/QuillPage/Wallet/Wallets/WalletDetail.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
```

Comment:
Lots of commented code in this file. Is it possible you intended these changes to be a draft?